app/controllers/api/v0/users_controller.rb

```diff
@@ -19,22 +19,20 @@ def create
 
   def find_or_create
     @user = User.find_by(email: params[:email], verified: 2)
-    unless @user
-      @user = User.create(email: params[:email], 
-                          verified: 2,
-                          username: User.username_generator, 
-                          password: "AccThruOAuth1", 
-                          password_confirmation: "AccThruOAuth1", 
-                          verification_token: nil)
-    end
+    @user ||= User.create(email: params[:email],
```

Comment:
Looks a lot better than the unless. (I still want the indents back on the lines below)